src/web/core/src/main/java/org/geoserver/web/GeoServerResourceStreamLocator.java

```diff
@@ -62,7 +65,9 @@ public IResourceStream locate(Class<?> clazz, String path) {
                         URL url = urls.nextElement();
 
                         try (InputStream in = url.openStream()) {
-                            properties.load(in);
+                            try (Reader reader = new InputStreamReader(in, StandardCharsets.UTF_8)) {
```

Comment:
Maybe  an issue in [JIRA](https://osgeo-org.atlassian.net/jira/software/c/projects/GEOS/issues/) is required?

Reply:
I should run `mvn -f src/pom.xml fmt:format` before `git commit`. Otherwise the java code formatter would complain.

Reply:
https://osgeo-org.atlassian.net/browse/GEOS-10189